packages/agent/conf/demos/knb/kvk/oid4vci_metadata/knb_kvk.json

```diff
@@ -49,11 +49,11 @@
             "background_color": "rgba(0, 0, 0, 0.2)",
             "text_color": "#FBFBFB",
             "logo": {
-              "url": "https://ibb.co/5Kjqr3j",
+              "url": "https://i.ibb.co/6Xt6HSt/kvk-logo.png",
               "alt_text": "KVK logo"
             },
             "background_image": {
-              "url": "https://ibb.co/L6DR6L8",
+              "url": "https://i.ibb.co/K7ch7MX/kvk-bg.png",
```

Comment:
this image has rounded corners. it should not have that. we should fix it, because i remember it looking bad in the demo.

check all images! cause i randomly picked this one to test and got lucky

Reply:
Place screenshot of this in DPP-84. Actually DPP-89 was only meant for DBC, but all these have to be fixed this sprint anyways.